migration/configuration.go

```diff
@@ -0,0 +1,26 @@
+package migration
+
+import (
+	"github.com/iov-one/weave/errors"
+	"github.com/iov-one/weave/gconf"
+)
+
+func init() {
```

Comment:
Great, we can unit test migrations with itself :)

Reply:
We are [almost there](https://www.youtube.com/watch?v=rc7-HbYvD9I)!